DEVELOPMENT.md

```diff
@@ -471,14 +471,14 @@ If you need to add a new CRD type, you will need to add:
     - [clusterrole-aggregate-edit.yaml](./config/clusterrole-aggregate-edit.yaml)
     - [clusterrole-aggregate-view.yaml](./config/clusterrole-aggregate-view.yaml)
 1. Add go structs for the types in
-    [pkg/apis/pipeline/v1alpha1](./pkg/apis/pipeline/v1alpha1) e.g
-    [condition_types.go](./pkg/apis/pipeline/v1alpha1/condition_types.go) This
+    [pkg/apis/pipeline/v1beta1](./pkg/apis/pipeline/v1beta1) e.g
+    [condition_types.go](./pkg/apis/pipeline/v1beta1/condition_types.go) This
     should implement the
-    [Defaultable](./pkg/apis/pipeline/v1alpha1/condition_defaults.go) and
-    [Validatable](./pkg/apis/pipeline/v1alpha1/condition_validation.go)
+    [Defaultable](./pkg/apis/pipeline/v1beta1/condition_defaults.go) and
```

Comment:
similarly, here please replace `condition_defaults.go` with `task_defaults.go`

And the following line, `condition_validation.go` to `task_validation.go`.

Reply:
Thank you for reviewing @pritidesai !
I just commit : [f71a32c](https://github.com/tektoncd/pipeline/pull/4256/commits/f71a32c42e5a1f1da6832caa1a4eadfa045cb103)